README.rst

```diff
@@ -207,3 +207,9 @@ There is a meeting every four weeks between Bodhi developers and stakeholder,
 held on IRC. If you would like to attend, you can see details here:
 
 https://apps.fedoraproject.org/calendar/meeting/4667/
+
+
+IRC
+===
+
+Come join us in the #bodhi channel on `Freenode <https://freenode.net/>`_.
```

Comment:
@pypingou I could alter it to be both. I could also just only put #fedora-apps. The only reason I made the #bodhi channel was that I recently learned that we have at least two users that are not Fedora (which is exciting!)

What is your preference?